packages/nocodb/src/lib/migrations/XcMigrationSourcev2.ts

```diff
@@ -10,6 +10,7 @@ import * as nc_019_add_meta_in_meta_tables from './v2/nc_019_add_meta_in_meta_ta
 import * as nc_020_kanban_view from './v2/nc_020_kanban_view';
 import * as nc_021_add_fields_in_token from './v2/nc_021_add_fields_in_token';
 import * as nc_022_qr_code_column_type from './v2/nc_022_qr_code_column_type';
+import * as nc_023_barcode_column_type from './v2/nc_023_barcode_column_type';
```

Comment:
023 has been used. You may update it when everything is reviewed and ready for merge.

Reply:
Yes I'm aware of it:) thank you